cogeo_mosaic/backends/base.py

```diff
@@ -78,10 +76,17 @@ class BaseBackend(BaseReader):
     def __attrs_post_init__(self):
         """Post Init: if not passed in init, try to read from self.input."""
         self.mosaic_def = self.mosaic_def or self._read()
-        self.minzoom = self.mosaic_def.minzoom
-        self.maxzoom = self.mosaic_def.maxzoom
         self.bounds = self.mosaic_def.bounds
 
+        mosaic_tms = self.mosaic_def.tilematrixset or WEB_MERCATOR_TMS
```

Comment:
in order to keep support for old mosaic document we assume the default TMS to be `WebMercatorQuad`